chapter_optimization/lr-scheduler.md

```diff
@@ -63,6 +63,54 @@ def train(net, train_iter, test_iter, num_epochs, loss, trainer, device):
           f'test acc {test_acc:.3f}')
 ```
 
+```{.python .input}
+#@tab tensorflow
+%matplotlib inline
+from d2l import tensorflow as d2l
+import tensorflow as tf
+import math
+from tensorflow.keras.callbacks import LearningRateScheduler
+
+def net():
+    return tf.keras.models.Sequential([
+        tf.keras.layers.Conv2D(filters=6, kernel_size=5, activation='sigmoid',
+                               padding='same'),
+        tf.keras.layers.AvgPool2D(pool_size=2, strides=2),
+        tf.keras.layers.Conv2D(filters=16, kernel_size=5,
+                               activation='sigmoid'),
+        tf.keras.layers.AvgPool2D(pool_size=2, strides=2),
+        tf.keras.layers.Flatten(),
+        tf.keras.layers.Dense(120, activation='sigmoid'),
+        tf.keras.layers.Dense(84, activation='sigmoid'),
```

Comment:
```suggestion
        tf.keras.layers.Dense(120, activation='relu'),
        tf.keras.layers.Dense(84, activation='relu'),
```

Reply:
Let the second last layer's activation be sigmoid. If not then some LR scheduler will fail to converge

Reply:
@astonzhang